apps/channels/signals.py

```
@@ -10,3 +12,23 @@ def invalidate_widget_session_cache(sender, instance: ExperimentSession, **kwarg
     """Invalidate widget session cache when session is updated."""
     if instance.external_id:
         delete_experiment_session_cached(instance.external_id)
+
+
+@receiver(post_save, sender=ChatMessage)
+def update_session_last_activity(sender, instance: ChatMessage, created, **kwargs):
```

Ruff (0.14.13)

Unused function argument: `sender`

(ARG001)